CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
+  * meta-pelion-edge itself as a yocto layer is now easier to incorporate with other layers, allowing other yocto projects to incorporate Pelion Edge.
+  * [recipe removals]
+    * wwrelay-utils recipe is removed.  Previously this bitbake recipe did many functions which are now replaced.  Reference the recipe additions section below.
+    * strace-plus recipe is removed as the image is no longer ships with developer tools enabled.
+    * tsb recipe is removed as the old wigwag relay is no longer supported.
+    * pps-tools recipe is removed as the image is no longer ships with developer tools enabled.
+    * node-znp recipe is removed as devicejs is deprecated.
+    * node-netkit recipe is removed as devicejs is deprecated.
+    * fftw recipe is removed as the image is no longer ships with developer tools enabled.
+    * emacs recipe is removed as the image is no longer ships with developer tools enabled.
+    * dnsmasq recipe is removed as Pelion Edge programs no longer need these features.
+    * deviceOSWD recipe is removed as the old wigwag relay is no longer supported.
+
+
+### Bug fixes
+- After production factory flow, if you run the info command before Edge core pairs with the cloud, the info command shows N/A for the deviceID while displaying connected. This has been fixed.
+ - Streamlined the startup sequence of Pelion Edge programs in order to remove the cyclic dependency, which caused many starts and stops of processes in certain conditions.
+
+### Known issues
+
+- When conducting back-to-back production factory flow with the Pelion Edge Provisioner, the mcc_config directory sometimes is not written correctly and upon reboot, Edge-Core does not connect properly. Workaround: Run the provisioner again.
+- Portal is not correctly updated after a firmware campaign in some instances.
+- [maestro] The FeatureMgmt config resource is initialized with max 3.8KB of file content. The remaining file content is truncated during initialization. This is most likely due to limitation of gorilla/websocket library, but needs further investigation. Although, Pelion DM users can still push a file size of max 64KB via cloud service APIs.
+
+### Limitations
+
+- The maximum translated devices behind the edge gateway is 100.
+- Devices behind Pelion Edge do not support [auto-observation.](https://www.pelion.com/docs/device-management/current/connecting/device-guidelines.html#auto-observation)
+- Pelion Device Management Client enabled devices must first boostrap to the Pelion Device Management Cloud before connecting to Pelion Edge.
+- No moving devices are supported. (Device would be moving from Edge to another Edge device.)
+
+### Important note
+
+While provisioning your gateway, please use `vendor-id=42fa7b48-1a65-43aa-890f-8c704daade54` to unlock the rich node features, such as gateway logs and gateway terminal in the Pelion web portal.
```

Comment:
Please mention that fota flag needs to be enabled while creating the build for this feature.

Reply:
@udayakumar17 its mentioned here - https://github.com/PelionIoT/meta-pelion-edge/pull/221/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR8 
We will also add that to the public doc, brought in by - https://github.com/PelionIoT/pelion-dm-edge-docs/pull/108